guides/user-accounts/_posts/2014-11-25-user-accounts.md

```diff
@@ -84,7 +84,7 @@ It’s important to keep your email address up to date on your Ona account, as t
 
 ##### <a name="upgrading-your-plan"></a>Upgrading your account
 
-We are accepting paid signups for the Ona service immediately. Signing up now gives you immediate access to a world class support team and qualifies you for an early adopter discount. To get started, visit the [upgrade page](https://ona.io/plans.html) and we will work with you to develop a pricing plan that meets your specific usage and support needs. 
+Anyone who signs up to Ona is automatically created as a Free plan user. The free plan is limited to 1 private project and 15 private forms. If you would wish to upgrade to one of our paid plans to increase the number or private projects/forms you can visit the [upgrade page](https://ona.io/plans.html) and we will work with you to develop a pricing plan that meets your specific usage and support needs. Upgrading to a paid plan also gives you direct access to Ona's world class support team. Features like [filtered datasets](https://help.ona.io/faq/filtered-datasets), Webhooks, [XLS Reports](https://help.ona.io/faq/what-is-a-xls-report) and [Ona Apps](https://help.ona.io/guides/onaapps/) are available for paid plans. 
```

Comment:
Suggestions:
- 1st sentence: change "Anyone who signs up to Ona automatically becomes a free plan user."
- 2nd sentence: include something about 500 submissions per month ...possibly with "...15 private forms with 500 submissions per month."
- 3rd sentence:  change "...number of private projects/forms, you can visit the upgrade page. We will work...."